async-request-reply/src/AsyncOperationStatusChecker.cs

```diff
@@ -75,16 +76,21 @@ public static async Task<IActionResult> Run(
                 }
             }
         }
-
-        private static async Task<IActionResult> OnCompleted(OnCompleteEnum OnComplete, BlockBlobClient inputBlob, string thisGUID)
+        private async Task<IActionResult> OnCompleted(OnCompleteEnum OnComplete, BlockBlobClient inputBlob, string thisGUID)
         {
             switch (OnComplete)
             {
                 case OnCompleteEnum.Redirect:
+
                     {
+                        //The typical way to generate a SAS token in code requires the storage account key.
+                        //If you need to use �Managed Identity� to control access to your storage accounts in code, which is something I highly recommend wherever possible as this is a security best practice.
+                        //In this scenario, you won�t have a storage account key, so you�ll need to find another way to generate the shared access signatures.
+                        //To do that, we need to use an approach called �user delegation� SAS . By using a user delegation SAS, we can sign the signature with the Azure Ad credentials instead of the storage account key.
```

Comment:
```suggestion
                        // In this scenario, you won't have a storage account key, so you'll need to find another way to generate the shared access signatures.
                        // To do that, we need to use an approach called user delegation SAS . By using a user delegation SAS, we can sign the signature with the Microsoft Entra ID credentials instead of the storage account key.
```